packages/x-charts/src/models/axis.ts

```diff
@@ -427,6 +428,34 @@ export type AxisValueFormatterContext<S extends ScaleName = ScaleName> =
       tickNumber?: number;
     };
 
+type MinMaxConfig<S extends ScaleName = ScaleName> = S extends ContinuousScaleName
+  ? S extends 'utc' | 'time'
+    ? {
+        /**
+         * The minimal value of the domain.
+         * If not provided, it gets computed to display the entire chart data.
+         */
+        min?: NumberValue;
+        /**
+         * The maximal value of the domain.
+         * If not provided, it gets computed to display the entire chart data.
+         */
+        max?: NumberValue;
+      }
+    : {
+        /**
+         * The minimal value of the domain.
+         * If not provided, it gets computed to display the entire chart data.
+         */
+        min?: number;
```

Comment:
Why do we want to make this distinction? Isn't any `NumberValue` acceptable when the scale isn't utc or time?

Reply:
Because there is no reason to put Dates into a log scale

Reply:
For me the goal is for user providing a date in the axis data with something else than a time or utc they should get a TS warning.

We don't gain anything by preventing them, but I don't see much more gain from allowing them